app/handlers/configHomography.py

```diff
@@ -10,17 +9,15 @@
 from ast import literal_eval
 from baseHandler import BaseHandler
 
-class UploadHomographyHandler(BaseHandler):
+class ConfigHomographyHandler(BaseHandler):
     """
-    @api {post} /uploadHomography/ Upload Homography
-    @apiName UploadHomography
-    @apiVersion 0.1.0
+    @api {post} /configHomography/ Config Homography
+    @apiName ConfigHomography
+    @apiVersion 0.2.0
     @apiGroup Upload
-    @apiDescription Use this route to upload homography files for a project.
+    @apiDescription Use this route to upload homography data for a project.
 
-    @apiParam {String} identifier The identifier of the project to upload files to.
-    @apiParam {File} aerial An aerial photo of the intersection.
-    @apiParam {File} camera A screenshot of the intersection from the video.
+    @apiParam {String} identifier The identifier of the project for which to configure the homography.
     @apiParam {Integer} unit_pixel_ratio The unit_pixel_ratio of the images (ie. 0.05 meters per pixel).
     @apiParam {JSON} aerial_pts A JSON array containing the coordinates of point clicks on the aerial image as arrays in the form [x_coord, y_coord]
```

Comment:
Technically its an Array of tuples but converting it to JSON is how its understood. I will change this to JSON Array to be specific.